ideascube/conf/base.py

```diff
@@ -298,6 +298,14 @@
         'description': _('Change the server name.'),
         'fa': 'info-circle',
     },
+    {
+        'is_staff': True,
+        'category': 'manage',
+        'url': 'server:cards',
+        'title': _('Cards'),
+        'description': _('Manage displayed cards'),
+        'fa': 'eye',
```

Comment:
This seems like the wrong metaphor to use here.

I'd use `th` or `th-large` instead.